### PR TITLE
Update Installation instructions in Quick Start Guides"

### PR DESCRIPTION
**Pull request by RukshiW**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? Yes
 - Ran FindSecurityBugs plugin and verified report? Yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? Yes

### CODE REVIEW

File: en/streaming-integrator/docs/quick-start-guide/getting-started-with-si.md [hunk 1]
```diff
@@ -15,18 +15,13 @@ In this guide, you will download the SI distribution, start it and then try out
...
 
-- On MacOS/Linux/CentOS, open a terminal and issue the following command:
-
-    `sudo wso2si`
-
-- On windows, go to **Start Menu -> Programs -> WSO2 -> Enterprise Integrator**. This opens a terminal. Start Streaming Integrator profile.
-
+Navigate to the `<SI_HOME>/bin` directory in the console and issue one of the following commands: <br/>
+    - For Windows: `server.bat`
+    - For Linux: `./serverg.sh`
```
Comment (Nirothipan) [inline en/streaming-integrator/docs/quick-start-guide/getting-started-with-si.md]: should this be server.sh

File: en/streaming-integrator/docs/quick-start-guide/quick-start-guide-101.md [hunk 2]
```diff
@@ -224,13 +225,9 @@ To deploy your Siddhi application in the Streaming Integrator server, follow the
...
     To deploy the Siddhi application, you need to run both the Streaming Integrator server and Streaming Integrator Tooling. The home directories of the Streaming Integrator server is referred to as `<SI_HOME>` and the home directory of Streaming Integrator Tooling is referred to as `<SI_TOOLING_HOME>`.
 
-1. Start the Streaming Integrator via one of the following methods depending on your operating system.
-
-   - On MacOS/Linux/CentOS, open a terminal and issue the following command:
-
-       `sudo wso2si`
-
-   - On windows, go to **Start Menu -> Programs -> WSO2 -> Enterprise Integrator**. This opens a terminal. Start Streaming Integrator profile.
+1. Start the Streaming Integrator server by navigating to the `<SI_HOME>/bin` directory from the CLI, and issuing one of the following commands:
```
Comment (Nirothipan) [inline en/streaming-integrator/docs/quick-start-guide/quick-start-guide-101.md]: Shouldn't we ask to issue the appropriate command as per their os. One of the commands doesn't sound to be correct.

File: en/streaming-integrator/docs/quick-start-guide/getting-started-with-si.md (hunk 1 above)
Comment (RukshiW) [reply en/streaming-integrator/docs/quick-start-guide/getting-started-with-si.md]: Changed as specified.

File: en/streaming-integrator/docs/quick-start-guide/quick-start-guide-101.md (hunk 2 above)
Comment (RukshiW) [reply en/streaming-integrator/docs/quick-start-guide/quick-start-guide-101.md]: Changed as specified.

Comment (Nirothipan) [inline en/streaming-integrator/docs/quick-start-guide/quick-start-guide-101.md]: this needs to be issue one of the appropriate commands , isn't it

File: en/streaming-integrator/docs/quick-start-guide/getting-started-with-si.md [hunk 3]
```diff
@@ -15,18 +15,13 @@ In this guide, you will download the SI distribution, start it and then try out
...
 
-- On MacOS/Linux/CentOS, open a terminal and issue the following command:
-
-    `sudo wso2si`
-
-- On windows, go to **Start Menu -> Programs -> WSO2 -> Enterprise Integrator**. This opens a terminal. Start Streaming Integrator profile.
-
+Navigate to the `<SI_HOME>/bin` directory in the console and issue one of the following commands depending on your operating system: <br/>
+    - For Windows: `server.bat`
+    - For Linux: `./server.sh`
```
Comment (Nirothipan) [inline en/streaming-integrator/docs/quick-start-guide/getting-started-with-si.md]: Linux / MacOS ?

Comment (Nirothipan) [reply en/streaming-integrator/docs/quick-start-guide/getting-started-with-si.md]: Please update in other places too

File: en/streaming-integrator/docs/quick-start-guide/quick-start-guide-101.md (hunk 2 above)
Comment (RukshiW) [reply en/streaming-integrator/docs/quick-start-guide/quick-start-guide-101.md]: Corrected.

File: en/streaming-integrator/docs/quick-start-guide/getting-started-with-si.md (hunk 3 above)
Comment (RukshiW) [reply en/streaming-integrator/docs/quick-start-guide/getting-started-with-si.md]: Corrected

**Pull request merged**